app/models/task.py

```diff
@@ -1,6 +1,29 @@
 from flask import current_app
 from app import db
+from app.models.goal import Goal
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, default=None, nullable=True)
+    is_complete = db.Column(db.Boolean, default=False)
```

Comment:
Another way to do this would be infer this value based off of the other columns instead of adding it as a column itself. We know that if `completed_at` is `NULL` (`None` when we're in Python), then the task must be incomplete.